packages/react-components/react-avatar/src/index.ts

```diff
@@ -5,8 +5,16 @@ export {
   useAvatarStyles_unstable,
   useAvatar_unstable,
 } from './Avatar';
-// eslint-disable-next-line deprecation/deprecation
-export type { AvatarNamedColor, AvatarProps, AvatarSlots, AvatarState, AvatarSizes, AvatarSize } from './Avatar';
+export type {
+  AvatarNamedColor,
+  AvatarProps,
+  AvatarSlots,
+  AvatarState,
+  // eslint-disable-next-line deprecation/deprecation
+  AvatarSizes,
+  AvatarSize,
+  AvatarShape,
```

Comment:
_nit:_ Keep alphabetical order

```suggestion
  AvatarShape,
  // eslint-disable-next-line deprecation/deprecation
  AvatarSizes,
  AvatarSize,
```